python/lsst/skymap/cellInfo.py

```diff
@@ -56,6 +62,32 @@ def getIndex(self):
         """
         return self._index
 
+    index = property(getIndex, None)
+
+    def getSequentialIndex(self):
+        """Return cell sequential index.
+
+        Returns
+        -------
+        result : `int`
+            Sequential cell index.
+        """
+        return self._sequentialIndex
+
+    sequentialIndex = property(getSequentialIndex, None)
```

Comment:
I have not yet looked at the PR in detail, but I wanted to make one point up-front based on what I knew was coming: I think we should make all of these new properties snake_case.  They're brand new, so there's no breakage from doing so, and there are no existing camelCase properties and relatively few camelCase public attributes, so the internal-to-this-package inconsistency wouldn't actually be too.  The new `cell_coadds` package that will make the most use of these will be snake_case throughout, so it'll be good to have more consistency with that, and of course it'd be much harder to change these anytime later.

One more side note: I think `property(getSequentialIndex)` should work too, and seems a bit better than the explicit `None`.

Reply:
I was wondering about this, and now it is done.